src/MagentoHackathon/Composer/Magento/Deploystrategy/Copy.php

```diff
@@ -71,6 +71,13 @@ public function createDelegate($source, $dest)
 
         // From now on $destPath can't be a directory, that case is already handled
 
+        // Check if the $desthPath is a symbolic link
+        if(is_link($destPath)){
+            echo "Target $dest already exists as sym link and will therefor not be updated\n";
+
+            return true;
```

Comment:
In case if we cannot finish a task because the environment we should not return `true`. An exception should be thrown instead. Also it highly undesirable to print something to the console at this point.